reports/report.md

```diff
@@ -7226,7 +7252,210 @@ State variable changes in this function but no event is emitted.
+
+	```solidity
+	    uint256 public variableValue;
+	```
+
+- Found in src/StateVariableCouldBeDeclaredImmutable.sol [Line: 6](../tests/contract-playground/src/StateVariableCouldBeDeclaredImmutable.sol#L6)
+
+	```solidity
+	    uint256 potentiallyImmutableUint;
+	```
+
+- Found in src/StateVariableCouldBeDeclaredImmutable.sol [Line: 9](../tests/contract-playground/src/StateVariableCouldBeDeclaredImmutable.sol#L9)
+
+	```solidity
+	    address potentiallyImmutableAddr;
+	```
+
+- Found in src/StorageConditionals.sol [Line: 5](../tests/contract-playground/src/StorageConditionals.sol#L5)
+
+	```solidity
+	    uint256 private s_sameConditionals;
+	```
+
+- Found in src/StorageConditionals.sol [Line: 6](../tests/contract-playground/src/StorageConditionals.sol#L6)
+
+	```solidity
+	    uint256 private s_differentConditionals;
+	```
+
+- Found in src/TestERC20.sol [Line: 7](../tests/contract-playground/src/TestERC20.sol#L7)
+
+	```solidity
+	    string public name;
+	```
+
+- Found in src/TestERC20.sol [Line: 8](../tests/contract-playground/src/TestERC20.sol#L8)
+
+	```solidity
+	    string public symbol;
+	```
+
+- Found in src/Trump.sol [Line: 128](../tests/contract-playground/src/Trump.sol#L128)
+
+	```solidity
+	    address payable private _taxWallet;
+	```
+
+- Found in src/Trump.sol [Line: 129](../tests/contract-playground/src/Trump.sol#L129)
+
+	```solidity
+	    address payable private _teamWallet;
+	```
+
+- Found in src/TxOriginUsedForAuth.sol [Line: 5](../tests/contract-playground/src/TxOriginUsedForAuth.sol#L5)
+
+	```solidity
+	    address public owner;
+	```
+
+- Found in src/UninitializedStateVariable.sol [Line: 9](../tests/contract-playground/src/UninitializedStateVariable.sol#L9)
+
+	```solidity
+	    uint256 public numPages; // GOOD (because it's initialized in constructor)
+	```
+
+- Found in src/UninitializedStateVariable.sol [Line: 37](../tests/contract-playground/src/UninitializedStateVariable.sol#L37)
+
+	```solidity
+	    uint256 public myVar; // initialized in extension, hence not captured
+	```
+
+- Found in src/UnprotectedInitialize.sol [Line: 7](../tests/contract-playground/src/UnprotectedInitialize.sol#L7)
+
+	```solidity
+	    address private owner;
+	```
+
+- Found in src/auditor_mode/ExternalCalls.sol [Line: 9](../tests/contract-playground/src/auditor_mode/ExternalCalls.sol#L9)
+
+	```solidity
+	    address private target;
+	```
+
+- Found in src/auditor_mode/ExternalCalls.sol [Line: 10](../tests/contract-playground/src/auditor_mode/ExternalCalls.sol#L10)
+
+	```solidity
+	    ExternalContractInterface private targetContract;
+	```
+
+- Found in src/eth2/DepositContract.sol [Line: 66](../tests/contract-playground/src/eth2/DepositContract.sol#L66)
+
+	```solidity
+	    bytes32[DEPOSIT_CONTRACT_TREE_DEPTH] zero_hashes;
+	```
+
+- Found in src/inheritance/InheritanceBase.sol [Line: 9](../tests/contract-playground/src/inheritance/InheritanceBase.sol#L9)
+
+	```solidity
+	    address public s_baseAddress;
```

Comment:
This should not be caught. If this were made immutable, the child contract would not compile.

Reply:
Good catch! I'll work on resolving this